buildSrc/src/main/resources/fips_java_bcjsse_11.policy

```diff
@@ -1,29 +0,0 @@
-/*
- * SPDX-License-Identifier: Apache-2.0
- *
```

Comment:
How can we test removing these files won't break dependencies? 

What is the impact of this change to end users, administrators, and other developers? Could you give a one line explanation for each of these? Thanks

Reply:
So basically this change removes legacy java8 files and merges java11 security/policy files into existing once in production code. I'm not sure how to deal with 'approved-mode' it's still in the making.

Reply:
While testing in FIPS-enabled mode, I realized the need to separate security files to differentiate between FIPS and non-FIPS environments. If you think we could also benefit from separating policy files, please let me know what changes I should make. 

In terms of what kind of impact the deleted file could have on the project as a whole - raises the question of whether we should proceed with a single-build solution or not (which has been discussed down below).